packages/office-addin-manifest/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "office-addin-manifest",
-  "version": "1.5.12",
```

Comment:
We shouldn't bump this version manually - it gets bumped automatically during publishing to NPM

Reply:
Gotcha, I'll change it back

Reply:
Fixed in latest commit